generators/app/templates/resource/tasks/config/webpack.config.js

```diff
@@ -0,0 +1,142 @@
+const webpack = require('webpack'),
+  path = require('path'),
+  ExtractTextPlugin = require('extract-text-webpack-plugin');
+
+module.exports = {
+  // Where to start
+  entry: {
+    TagInput: path.resolve(__dirname, '../../src/index.js')
+  },
+
+  // Where to output
+  output: {
+    path: path.resolve(__dirname, '../../dist'),
+    filename: '[name].js',
+    libraryTarget: 'commonjs2'
+  },
+
+  externals: {
+    classnames: true,
+    react: 'React',
+    'react-dom': true,
+    'ship-components-outsideclick': true,
+    'ship-components-icon': true,
+    'react-addons-css-transition-group': true
+  },
+
+  module: {
+    rules: [
+      {
+        test: /\.(jsx?|es6)$/,
+        enforce: 'pre',
+        exclude: /(node_modules|dist)/,
+        include: /src\/.*/,
+        use: 'eslint-loader'
+      },
+      // ES6/JSX for App
+      {
+        test: /\.(jsx?|es6)$/,
+        exclude: [
+          /node_modules/
+        ],
+        use: 'babel-loader'
+      },
+      {
+        test: /\.(jsx?|es6)$/,
+        include: [
+          /ship-components-.*\/src/
+        ],
+        use: 'babel-loader'
+      },
+      {
+        test: /\.(png|woff|woff2|eot|ttf|svg)(\?v=[0-9]\.[0-9]\.[0-9])?$/,
+        use: [
+          {
+            loader: 'file-loader',
+            options: {
+              name: '[path][name].[ext]'
+            }
+          }
+        ]
+      },
+      // CSS Modules
+      {
+        test: /\.css$/,
+        use: ExtractTextPlugin.extract({
+          fallback: 'style-loader',
+          use: [
+            {
+              loader: 'css-loader',
+              options: {
+                modules: true,
+                importLoaders: 1,
+                localIdentName: '[name]--[local]'
+              }
+            },
+            {
+              // CSS Modules
+              loader: 'postcss-loader',
+              options: {
+                plugins: () => [
+                  require('postcss-nested')(),
```

Comment:
Are these the standard plugins? We're making an assumption here.

Reply:
Not sure what you mean by standard plugins!! These are the ones we use on almost all of our components.